### DISCUSSION TITLE
v3中阿里云webdav出现大量403

### DISCUSSION THREAD

**Discussion by yqchilde** [AlistGo/alist#1724] [Invalid question]
有没有遇到同样问题的朋友，403读不到资源，rclone配置都一样
v3:
![image](https://user-images.githubusercontent.com/36953759/191060984-56264fc3-d823-42c0-be78-fbe0b7e2ce08.png)

v2:
![image](https://user-images.githubusercontent.com/36953759/191061012-d88a03be-ec1c-4509-803b-363c895b28e1.png)

**Comment by yqchilde** [comment]
已解决

**Reply by wuerbuzuo** [reply]
怎么解决的？一样遇到了，用其他的都没问题就只有alist的有问题

**Reply by yqchilde** [reply]
> 怎么解决的？一样遇到了，用其他的都没问题就只有alist的有问题

webdav策略改为本地代理，原因：webdav不支持重定向

**Comment by coderhwz** [comment]
把报错文字发一下，方便后来者能顺利搜索到这个报错：

`vfs cache: failed to download: vfs reader: failed to write to cache file: 403 Forbidden`

**Comment by Kiri256** [comment]
感谢

**Comment by a510736676** [comment]
怎么修改webdav策略呀

**Reply by perfectnewer** [reply]
在对应存储的设置里面改
